<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="num1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OPERANDO 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="num2_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OPERANDO 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="resul_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESULTADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="real_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="ieee_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IEEE754</string>
    </property>
   </widget>
   <widget class="QLabel" name="hex_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HEXADECIMAL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1Real">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2Real">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1IEEE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2IEEE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1Hex">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2Hex">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="resulReal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="resulIEEE">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="resulHex">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>350</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>911</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="add">
       <property name="minimumSize">
        <size>
         <width>299</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>299</width>
         <height>61</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mult">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>61</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="div">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>61</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
